lib/msal-core/README.md

```diff
@@ -17,25 +17,56 @@ The MSAL library for JavaScript enables client-side JavaScript web applications,
 
 ### Via Latest Microsoft CDN Version:
 
-    <!-- Latest compiled and minified JavaScript -->
-    <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.1.3/js/msal.js"></script>
-    <script type="text/javascript" src="https://alcdn.msauth.net/lib/1.1.3/js/msal.min.js"></script>
-    
-    <!-- Alternate region URLs -->
-    <script type="text/javascript" src="https://alcdn.msftauth.net/lib/1.1.3/js/msal.js"></script>
-    <script type="text/javascript" src="https://alcdn.msftauth.net/lib/1.1.3/js/msal.min.js"></script>
+#### Latest compiled and minified JavaScript
```

Comment:
Pulling dev into `multiple-instances` i think would help clean up this diff.

Reply:
I did already. Not sure why this diff is showing.

Reply:
I pulled dev into both the branches. :). I think there is a local and remote state mismatch in git on my end (I use github desktop), I can try this differently.